server/models/pattern/core/pattern.go

```diff
@@ -255,6 +284,13 @@ func NewPatternFileFromCytoscapeJSJSON(name string, byt []byte) (Pattern, error)
 		}
 		eleToSvc[ele.Data.ID] = svc.Name //will be used while adding depends-on
 		pf.Services[svc.Name] = &svc
+
+		// clean some redundant fields from the settings
+		delete(svc.Settings, "name")
+		delete(svc.Settings, "namespace")
```

Comment:
Why to put these here in the first place?

Reply:
Lets not add these fields inside of `settings`

Reply:
remove redudancy.
the name, namespace, labels and annotations were in two places with the exact same value being duplicated.

Reply:
I may have get your next message after writing my first.